amplipi/rt.py

```diff
@@ -523,15 +523,15 @@ def update_zone_vol(self, zone, vol):
 
       Args:
         zone: zone to adjust vol
-        vol: int in range[MIN_VOL, 0]
+        vol: int in range[MIN_VOL_DB, 0]
```

Comment:
Can this be an F-string?

Reply:
Not without a decorator, plus I'd argue that would mask how to properly use this method.